lib/frameworks/utils.rb

```diff
@@ -47,6 +47,13 @@ def check_expected_section_items(sections, table)
       end
     end
 
+    def check_expected_section_item(section, table)
```

Comment:
@pratyush23 It would be good, if you comment above this method for the action it performs.
As the method at https://github.com/bbc/frameworks-capybara/blob/master/lib/frameworks/utils.rb#L41 do not have much difference in name.